android/build.gradle

```
@@ -58,7 +58,7 @@ android {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     // Customer.io SDK
-    def cioVersion = "[3.3,4.0)"
+    def cioVersion = "[3.4,4.0)"
```

Since this is range, we can probably update the lowest version to current latest? 🤔 

```suggestion
    def cioVersion = "[3.4.2,4.0)"
```